wallet/src/de/schildbach/wallet/ui/transactions/TransactionGroupDetailsFragment.kt

```diff
@@ -52,11 +53,19 @@ class TransactionGroupDetailsFragment() : OffsetDialogFragment(R.layout.transact
 
         binding.dashAmount.setFormat(viewModel.dashFormat)
 
-        if (transactionWrapper is FullCrowdNodeSignUpTxSet) {
-            binding.groupTitle.text = getString(R.string.crowdnode_account)
-            binding.icon.setImageResource(R.drawable.ic_crowdnode_logo)
-            binding.detailsTitle.text = getString(R.string.crowdnode_tx_set_title)
-            binding.detailsMessage.text = getString(R.string.crowdnode_tx_set_explainer)
+        when (transactionWrapper) {
+            is FullCrowdNodeSignUpTxSet -> {
+                binding.groupTitle.text = getString(R.string.crowdnode_account)
+                binding.icon.setImageResource(R.drawable.ic_crowdnode_logo)
+                binding.detailsTitle.text = getString(R.string.crowdnode_tx_set_title)
+                binding.detailsMessage.text = getString(R.string.crowdnode_tx_set_explainer)
+            }
+            is CoinJoinMixingTxSet -> {
+                binding.groupTitle.text = getString(R.string.coinjoin_mixing_transactions)
+                binding.icon.setImageResource(R.drawable.ic_coinjoin)
+                binding.detailsTitle.text = getString(R.string.crowdnode_tx_set_title)
+                binding.detailsMessage.text = getString(R.string.coinjoin_transaction_group)
+            }
```

Comment:
At some point we should move it to TxResourceMapper too to keep to open-closed principle